R/workflow.R

```diff
@@ -14,13 +17,14 @@
 ##'
 ##' # To run most reports, provide the report name (and the path if
 ##' # not running in the working directory, as is the case here):
-##' #id <- orderly::orderly_workflow("my_workflow", root = path)
-orderly_workflow <- function(name, root = NULL, locate = TRUE) {
+##' ids <- orderly::orderly_workflow("my_workflow", root = path)
+orderly_workflow <- function(name, envir = NULL, root = NULL, locate = TRUE,
+                             message = NULL, instance = NULL, remote = NULL) {
   ## Locate file
   config <- orderly_config_get(root, locate)
   workflow <- orderly_workflow_get(name, config)
 
-  workflow$run()
+  workflow$run(envir, message, instance, remote)
 
   ## TODO: Write to DB after run
```

Comment:
I think the DB work will follow pretty easily after #232 and we can (and should) do this one by one as we go.